### DISCUSSION TITLE
Sonarr/Radarr 'URL Base' option resets with each update

### DISCUSSION THREAD

**Discussion by celynw** [alexbelgium/hassio-addons#810] [Q&A] [answered]
Every time the Sonarr or Radarr addon is updated through Home Assistant, it repopulates the 'URL Base' field back to `sonarr/` or `radarr/` respectively. This is a bit strange, because it says right underneath the field that the default is empty.
It causes a problem other addons (such as Autobrr, also from this repository), as the endpoint address isn't where it's expected to be. I clear it and restart the addon every time I notice.

The setting is located in `Settings` -> `General` -> `Host` -> `URL Base`.

![image](https://user-images.githubusercontent.com/3299161/234040234-b3b5dd49-a522-4b5f-ab9e-060d0caa8eb7.png)

It might _seem_ like a problem with Sonarr or Radarr, but I've run those directly from docker images for months before, and this never happened.

I haven't raised this as a bug, because I thought it could be being overridden by some config file somewhere, but I'm not sure where. It's not an option in the addon config.

Any ideas?

**Comment by alexbelgium** [comment, marked as answer]
Hi, indeed i've forced this behavior as this is required to allow ingress to work... Is that an issue for you ? Can't you "just" refer to this base url in autobrr ?

**Reply by celynw** [reply]
Ah ok, I think I see where you mean now

https://github.com/alexbelgium/hassio-addons/blob/140f7f3f47cfc212b3c41e64e8527b9a2362bf08/sonarr/rootfs/etc/nginx/servers/ingress.conf#L26

I suppose I could, since there's a good reason why it's like this. I have a few other services (mobile apps, uptime checking) but I think they should be fine. The only real casualty is my OCD for having redundancy in my URL!

Thanks for the very quick feedback.
